### PR TITLE
Adding matrixprofile 1.1.10

### PR DESCRIPTION
**Pull request by vanbenschoten**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licxenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by vanbenschoten** [comment]
Confirming that I'm willing to be listed here.

**Comment by tylerwmarrs** [comment]
Please list me here.

**Comment by vanbenschoten** [comment]
@conda-forge/staged-recipes Hello! I realize this PR took a bit of time to put together. Is there anything else we need to do now that all checks have passed?

File: recipes/matrixprofile/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+numpy:
```
Comment (chrisburr) [inline recipes/matrixprofile/conda_build_config.yaml:1]: I don't think this file is needed

File: recipes/matrixprofile/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - cython
+    - numpy 1.16.*
```
Comment (chrisburr) [inline recipes/matrixprofile/meta.yaml:25]:
```suggestion
    - numpy
```

**Pull request merged**